src/Microsoft.ML.ImageAnalytics/VectorToImageTransform.cs

```diff
@@ -108,29 +119,289 @@ public class Arguments : TransformInputBase
             public int ImageHeight;
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Offset (pre-scale)")]
-            public Single? Offset;
+            public Single Offset = Defaults.Offset;
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Scale factor")]
-            public Single? Scale;
+            public Single Scale = Defaults.Scale;
+        }
+
+        internal static class Defaults
+        {
+            public const ImagePixelExtractingEstimator.ColorBits Colors = ImagePixelExtractingEstimator.ColorBits.Rgb;
+            public const bool InterleaveArgb = false;
+            public const Single Offset = 0;
+            public const Single Scale = 1;
+        }
+
+        internal const string Summary = "Converts vector array into image type.";
+        internal const string UserName = "Vector To Image Transform";
+        internal const string LoaderSignature = "VectorToImageConverter";
+        private static VersionInfo GetVersionInfo()
+        {
+            return new VersionInfo(
+                modelSignature: "VECTOIMG",
+                //verWrittenCur: 0x00010001, // Initial
+                //verWrittenCur: 0x00010002, // Swith from OpenCV to Bitmap
+                verWrittenCur: 0x00010003, // don't serialize sizeof(Single)
+                verReadableCur: 0x00010003,
```

Comment:
>0x00010003 [](start = 32, length = 10)

You can always have check for model version and read one Int32 from stream if it's old one.

I have separate PR where I bump version for ImagePixelExtractor (I introduce colors order). So I plan do the same for VectorToImage and bump version again.
I can get rid of single and add order in same time.
Does it sound good for you?
If you still want to get rid of single in this iteration, I would suggest to keep old models readable.